Automattic/wp-calypso
# Jetpack Onboarding: Add new illustrations

This PR introduces the new illustrations that will be used in the Jetpack Onboarding flows. Borrowed from #17094, but moved to separate svg assets, cleaned up and optimized.

The following illustrations are being introduced:

**Site Type - Personal:**
![](https://cldup.com/AJ9scfGsoS.png)

**Site Type - Business**
![](https://cldup.com/ZoMEwva7DW.png)

**Contact Us**
![](https://cldup.com/_Dldl6HBHl.png)

**Home Page - Static**
![](https://cldup.com/GT8Rg5ABFQ.png)

**Home Page - News(blog)**
![](https://cldup.com/2-DR6UmL4O.png)

To test:
* Load each of those in an SVG viewer or your browser.
* Verify they look as expected.
* Verify they're optimal for production.
